packages/zipkin-context-cls/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "zipkin-context-cls",
+  "version": "0.1.9",
+  "description": "A Context API implementation that uses continuation-local-storage under the hood",
+  "main": "src/CLSContext.js",
+  "scripts": {
+    "test": "node ../../node_modules/mocha/bin/mocha --require ../../test/helper.js"
+  },
+  "author": "Openzipkin <openzipkin.alt@gmail.com>",
```

Comment:
My fault; it's Openzipkin all over zipkin-js now. We can just do a search-replace later on.